content/100-getting-started/index.mdx

```diff
@@ -1,108 +1,169 @@
 ---
-title: 'Get started'
-metaTitle: 'Get started'
-metaDescription: 'Get started'
+title: 'Get Started'
+metaTitle: 'Get started with Prisma'
```

Comment:
Shouldn't this be 'Get Started with Prisma ORM'

Reply:
Since this page also gives avenues to get started with Accelerate and Pulse, I think "Get started with Prisma" is more appropriate.